tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/record/OpenRecordNegativeTest.java

```diff
@@ -58,6 +58,8 @@ public void testInvalidRestField() {
         validateError(result, indx++,
                       "invalid operation: type 'Person' does not support optional field access for field 'firstName'",
                       58, 26);
+        validateError(result, indx++, "incompatible types: expected 'record {| int i; record {| string name;" +
```

Comment:
This is not the ideal place for this test right - since it is `testInvalidRestField()`?

Reply:
Moved to a separate method (code snippet to a separate file)